app/views/projects/show.html.erb

```diff
@@ -0,0 +1,7 @@
+<h1><%= @project.name %></h1>
+<p>Material: <%= @project.material %></p>
+<h3>Challenge theme: <%= @project.challenge.theme %> </h3>
+<h3>Number of Contestants: <%=@project.contestants.count %></h3>
+<h3>Average Contestant Experience: <%=@project.contestants.average(:years_of_experience).to_s %></h3>
```

Comment:
This is also an MVC infraction, since we want to only do AR calculations like this in the model, and test it before we show the user. It's good AR and I'm glad to see you're using the `.average` method, but it really should go in the model as a method, and then we should test that method to be sure we're returning the correct values. This would be the 2nd MVC infraction I'm seeing.